tests/expectations/core/test_expect_column_max_to_be_between.py

```diff
@@ -1,51 +0,0 @@
-from datetime import datetime
-
-import pandas as pd
-import pytest
-
-from great_expectations.core.batch import RuntimeBatchRequest
-from great_expectations.data_context import DataContext
-
-
-@pytest.mark.filesystem
-def test_expect_column_max_to_be_between_warn_parse_strings_as_datetimes(
-    data_context_with_datasource_pandas_engine,
-):
```

Comment:
All these tests test for a path we no longer support - just deleting them outright